#  author John Dey, fix incorrect paths for ref data. Update file names for ref data
diff -ruN CRISPResso2-2.3.1.orig/tests/FANC.batch CRISPResso2-2.3.1/tests/FANC.batch
--- CRISPResso2-2.3.1.orig/tests/FANC.batch	2024-05-13 08:41:32.000000000 -0700
+++ CRISPResso2-2.3.1/tests/FANC.batch	2024-06-14 16:23:37.616827000 -0700
@@ -1,3 +1,3 @@
 n	r1
-Untreated	tests/FANC.Untreated.fastq
-Cas9	tests/FANC.Cas9.fastq
+Untreated	FANC.Untreated.fastq
+Cas9	FANC.Cas9.fastq
diff -ruN CRISPResso2-2.3.1.orig/tests/testRelease.sh CRISPResso2-2.3.1/tests/testRelease.sh
--- CRISPResso2-2.3.1.orig/tests/testRelease.sh	2024-05-13 08:41:32.000000000 -0700
+++ CRISPResso2-2.3.1/tests/testRelease.sh	2024-06-14 15:04:55.646070000 -0700
@@ -15,20 +15,20 @@
 CRISPRessoWGS -b Both.Cas9.fastq.smallGenome.bam -r smallGenome/smallGenome.fa -f Cas9.regions.txt --debug &> CRISPRessoWGS_on_Both.Cas9.fastq.smallGenome.log
 
 echo TESTING CRISPRESSO2
-diff CRISPResso_on_FANC.Cas9/Reference.nucleotide_frequency_table.txt expectedResults/CRISPResso_on_FANC.Cas9/Reference.nucleotide_frequency_table.txt
-diff CRISPResso_on_FANC.Cas9/CRISPResso_quantification_of_editing_frequency.txt tests/expectedResults/CRISPResso_on_FANC.Cas9/CRISPResso_quantification_of_editing_frequency.txt
+diff CRISPResso_on_FANC.Cas9/Nucleotide_frequency_table.txt expectedResults/CRISPResso_on_FANC.Cas9/Nucleotide_frequency_table.txt
+diff CRISPResso_on_FANC.Cas9/CRISPResso_quantification_of_editing_frequency.txt expectedResults/CRISPResso_on_FANC.Cas9/CRISPResso_quantification_of_editing_frequency.txt
 
 echo TESTING CRISPRESSO2 PARAMS
-diff CRISPResso_on_params/FANC.nucleotide_frequency_table.txt expectedResults/CRISPResso_on_params/FANC.nucleotide_frequency_table.txt
-diff CRISPResso_on_params/CRISPResso_quantification_of_editing_frequency.txt tests/expectedResults/CRISPResso_on_params/CRISPResso_quantification_of_editing_frequency.txt
+diff CRISPResso_on_params/FANC.Nucleotide_frequency_table.txt expectedResults/CRISPResso_on_params/FANC.Nucleotide_frequency_table.txt
+diff CRISPResso_on_params/CRISPResso_quantification_of_editing_frequency.txt expectedResults/CRISPResso_on_params/CRISPResso_quantification_of_editing_frequency.txt
 
 echo TESTING BATCH
-diff CRISPRessoBatch_on_FANC/Reference.MODIFICATION_FREQUENCY_SUMMARY.txt tests/expectedResults/CRISPRessoBatch_on_FANC/Reference.MODIFICATION_FREQUENCY_SUMMARY.txt
+diff CRISPRessoBatch_on_FANC/MODIFICATION_FREQUENCY_SUMMARY.txt expectedResults/CRISPRessoBatch_on_FANC/MODIFICATION_FREQUENCY_SUMMARY.txt
 
 echo TESTING POOLED
-diff CRISPRessoPooled_on_Both.Cas9/SAMPLES_QUANTIFICATION_SUMMARY.txt tests/expectedResults/CRISPRessoPooled_on_Both.Cas9/SAMPLES_QUANTIFICATION_SUMMARY.txt
+diff CRISPRessoPooled_on_Both.Cas9/SAMPLES_QUANTIFICATION_SUMMARY.txt expectedResults/CRISPRessoPooled_on_Both.Cas9/SAMPLES_QUANTIFICATION_SUMMARY.txt
 
 echo TESTING WGS
-diff CRISPRessoWGS_on_Both.Cas9.fastq.smallGenome/SAMPLES_QUANTIFICATION_SUMMARY.txt tests/expectedResults/CRISPRessoWGS_on_Both.Cas9.fastq.smallGenome/SAMPLES_QUANTIFICATION_SUMMARY.txt
+diff CRISPRessoWGS_on_Both.Cas9.fastq.smallGenome/SAMPLES_QUANTIFICATION_SUMMARY.txt expectedResults/CRISPRessoWGS_on_Both.Cas9.fastq.smallGenome/SAMPLES_QUANTIFICATION_SUMMARY.txt
 
 echo Finished
